src/main.rs

```diff
@@ -85,28 +85,26 @@ fn main() {
         // If we got here it means the IP was not found within the HashMap, this
         // means the scan couldn't find any open ports for it.
 
-        let x = format!("{} Looks like I didn't find any open ports for {:?}. This is usually caused by a high batch size.
-        \n*I used {} batch size, consider lowering to {} with {} or a comfortable number for your system.
+        let x = format!("Looks like I didn't find any open ports for {:?}. This is usually caused by a high batch size.
+        \n*I used {} batch size, consider lowering it with {} or a comfortable number for your system.
         \n Alternatively, increase the timeout if your ping is high. Rustscan -t 2000 for 2000 milliseconds (2s) timeout.\n",
-        "ERROR",
         ip,
         opts.batch_size,
-        (opts.batch_size / 2).to_string(),
         "'rustscan -b <batch_size> <ip address>'");
         warning!(x, opts.quiet);
     }
 
     for (ip, ports) in ports_per_ip.iter_mut() {
         let nmap_str_ports: Vec<String> = ports.into_iter().map(|port| port.to_string()).collect();
 
-        detail!("Starting Nmap", opts.quiet);
+        detail!("Starting Nmap", opts.quiet || opts.no_nmap);
```

Comment:
WOAH! I wrote this macro and I never realised you could just pas in bools like that. :exploding_head: